Jenkinsfile.nightly

```diff
@@ -18,18 +18,13 @@ pipeline {
 			steps {
 				sh '''
 				sudo service postgresql restart
-				cp test/data/config.json test/data/genesis_block.json .
+				cp config/devnet/config.json config/devnet/genesis_block.json .
```

Comment:
Why we need to do that?

Reply:
Because the config now is in another path... ??

Reply:
I mean why we need to copy those to main directory.

Reply:
You're right. Probably it isn't needed anymore. I'll remove that command and check the nightly build again.